src/WebApp/flask/spark/customScripts/jupyter.sh

```diff
@@ -52,6 +52,55 @@ if  [ "$IS_MASTER" = "1" ]; then
 }
 EOF
 
+
+    cat << EOF > $SPARK_HOME/conf/hive-site.xml
```

Comment:
why is this combined with another pull request?

Reply:
Did not do a git merge. Will remove this one